<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticalAnalysisChartsWidget</class>
 <widget class="QWidget" name="StatisticalAnalysisChartsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1347</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1327</width>
        <height>597</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayoutChart">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayoutYMinMax">
     <item row="0" column="1">
      <widget class="QLabel" name="labelMeasureYMin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Y-Axis Min:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxYMax">
       <property name="maximum">
        <double>1000000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacerChartMinMax">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="commandExportChart">
       <property name="text">
        <string>Export...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxYMin">
       <property name="maximum">
        <double>1000000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="labelMeasureYMax">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Y-Axis Max:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBoxSetYMinMax">
       <property name="text">
        <string>Set:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayoutMeasure">
     <item>
      <widget class="QLabel" name="labelMeasure">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Measure:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxMeasure"/>
     </item>
     <item>
      <widget class="QLabel" name="labelGroupByCom">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Group by Metadata: Communication:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxGroupByCom"/>
     </item>
     <item>
      <widget class="QLabel" name="labelGroupBySpk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Speaker:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxGroupBySpk"/>
     </item>
     <item>
      <widget class="QPushButton" name="commandDrawChart">
       <property name="text">
        <string>Draw Chart</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
